packages/sslconf/sslconf.0.8.3/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+maintainer: "Tony Wuersch <tony.wuersch@gmail.com>"
+homepage: "https://github.com/awuersch/sslconf"
+dev-repo: "https://github.com/awuersch/sslconf.git"
+bug-reports: "https://github.com/awuersch/sslconf/issues"
+doc: "https://awuersch.github.io/sslconf/doc"
+authors: [
+  "Tony Wuersch <tony.wuersch@gmail.com>"
+]
+license: "ISC"
+depends: [
+  "jbuilder" {build}
+  "ppx_sexp_conv" {build}
+  "sexplib"
+  "astring"
+  "rresult"
+  "fpath"
+  "cmdliner"
+  "topkg-jbuilder" {build}
+  "ounit" {test}
+  "odoc" {doc}
+]
+build: [
+  ["jbuilder" "subst"] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+available: [ ocaml-version >= "4.03.0" & ocaml-version < "4.06.0" ]
```

Comment:
This all looks very good thanks for your patience. 

Once last nit/question, is there any reason why you constrained this to `< "4.06.0"` ? The current policy of the opam-repository is to have unconstrained upper bounds and set them on new releases that break the package.